src/AppForeach.Framework.Hosting/Features/Sql/SqlMigrationStartup.cs

```diff
@@ -13,28 +15,38 @@ public class SqlMigrationStartup<TDbContext> : IApplicationStartup
         where TDbContext : DbContext
     {
         private readonly IOptions<SqlMigrationOptions<TDbContext>> migrationOptions;
+        private readonly IServiceLocator serviceLocator;
 
-        public SqlMigrationStartup(IOptions<SqlMigrationOptions<TDbContext>> migrationOptions)
+        public SqlMigrationStartup(IOptions<SqlMigrationOptions<TDbContext>> migrationOptions, IServiceLocator serviceLocator)
         {
             this.migrationOptions = migrationOptions;
+            this.serviceLocator = serviceLocator;
         }
 
         public async Task Run(CancellationToken cancellationToken)
         {
             var optionsBuilder = new DbContextOptionsBuilder<TDbContext>();
             optionsBuilder.UseSqlServer(migrationOptions.Value.ConnectionString, migrationOptions.Value.DbContextOptions);
```

Comment:
The service locator is needed in the DBContext to get entity specifications, like:
![image](https://github.com/user-attachments/assets/5440bf7e-a9ce-4920-ab28-fc312991b934)